### PR TITLE
Made it explicit that the QRG is not part of the standard, but that i…

### PR DESCRIPTION
**Pull request by tucotuco**
…t draws on the standard and adds comments and examples.

Addresses two check points in issue #254

### CODE REVIEW

**Comment by baskaufs** [comment]
With respect to the comments and examples: it might be better to say that they are not *normative* parts of the standard rather than that they aren't part of the standard. I realize that throwing around jargon like "normative" isn't in the user-friendly spirit of the QRG, but those metadata fields (comments and examples) do appear along with the other metadata fields when the vocabulary metadata are provided.

**Comment by tucotuco** [comment]
Is that (68fcf71) better?

**Comment by tucotuco** [comment]
Updated the original comment above, I incorrectly reference Issue 256, but have now amended it to #254

**Comment by baskaufs** [comment]
Marvelous!

File: build/terms.tmpl [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+This document is intended to be an easy-to-read reference of the currently recommended terms maintained as part of the [Darwin Core standard](https://www.tdwg.org/standards/dwc/). This page itself is not part of the standard. It draws on the term names and definitions from the normative part of the standard and combines them with comments and examples that are not normative, but that are meant to help people to use the terms consistently. Categories such as `Occurrence` and `Event` correspond to Darwin Core classes, which are special category terms used group sets of terms for convenience. [Files with lists of these terms](https://github.com/tdwg/dwc/tree/master/dist) and [their full history](https://github.com/tdwg/dwc/blob/master/vocabulary/term_versions.csv) can be found in the [Darwin Core repository](https://github.com/tdwg/dwc).
```
Comment (peterdesmet) [inline build/terms.tmpl:11]: ... used **to** group sets ...

**Pull request merged**